### DISCUSSION TITLE
[gatsby-theme-cara] Using an image as background for Project Cards

### DISCUSSION THREAD

**Discussion by Antonio-Riccelli** [LekoArts/gatsby-themes#826] [Questions]
I would like to use an image as background, instead of a gradient, but I'm struggling to understand how to customise it. 

Any help is appreciated. Thank you.

```
<ProjectCard
  title="Setting  up a yarn Workspace"
  link="https://www.lekoarts.de/javascript/setting-up-a-yarn-workspace-with-typescript-eslint-and-cypress?utm_source=cara&utm_medium=Theme"
  bg="linear-gradient(to right, #662D8C 0%, #ED1E79 100%)"
>

**Comment by LekoArts** [comment]
You'll want to shadow (https://github.com/LekoArts/gatsby-themes/tree/main/themes/gatsby-theme-cara#shadowing) the https://github.com/LekoArts/gatsby-themes/blob/main/themes/gatsby-theme-cara/src/components/project-card.tsx component, then you can use any props/things you like.
